cad/gtkwave/Portfile

```diff
@@ -19,17 +19,17 @@ long_description \
 homepage        http://gtkwave.sourceforge.net
 master_sites    sourceforge:project/gtkwave/gtkwave-${version}
 
-checksums       rmd160  8299c47c7d8a42f26e6d8a402df0ee3cfe2fe56d \
-                sha256  570e7f295a35cdd5ba173d4e3b6adb29e78354e76cc21a7ce2932551863e7bb9 \
-                size    3511620
+checksums       rmd160  7e1556072540ab5ef1b94dc2a823533ba6ba15ba \
+                sha256  55520fc308244c5dc99d5a3f42f5e782eb8e6a9e81cece5c84ea3f11875bff13 \
+                size    3506108
 
 depends_build-append \
                 port:pkgconfig
 
 depends_lib-append \
                 port:bzip2 \
                 path:lib/pkgconfig/gtk+-2.0.pc:gtk2 \
-                port:gtk-osx-application-gtk2 \
+                port:gtk-osx-application-gtk3 \
```

Comment:
this doesn't make sense to me. A few lines above you still depend on "gtk2" so why is this now depending on "*-gtk3"? I would think that should be consistent: either gtk2 or gtk3.